src/standardMethods.ts

```diff
@@ -11,24 +11,27 @@ import { MetadataMap } from './metadata';
 import { promiseAllWithData } from './resources/assets/assetUtils';
 import { CursorResponse, ItemsResponse } from './types/types';
 
-type CreateEndpoint<RequestType, ResponseType> = (
-  items: RequestType[]
-) => Promise<ResponseType[]>;
+export type Newable<T> = new (...args: any[]) => T;
```

Comment:
you never use it. remove?

Reply:
It was something that were copied from the files Fredrik had already started on, so I don't know what it does.

Reply:
yeah, but if he needs it he will probably add it to his pr